api/interestr/website/templates/website/login.html

```diff
@@ -17,25 +17,30 @@
     <div class="container">
       <div class="col-md-6">
         <h2>Login</h2>
+		{% if err_msg %}
+		<div style=>
+		<h4 style="color:Red">Invalid Credentials!</h4>
+		</div>
+		{%endif %}
         <hr>
         <form method='POST'>
-
-        {% csrf_token %}
+			{% csrf_token %}
+        <!-- 
 
         {{form.as_p}}
         <button type="submit" class="btn btn-warning">Submit</button>
-
-        <!--
+        -->
+       
           <div class="form-group">
-            <label for="email">Username/Email:</label>
-            <input type="email" class="form-control" id="email" placeholder="Username or email address">
+            <label for="username_or_email">Username/Email:</label>
+            <input type="text" class="form-control" name="username_or_email" placeholder="Username or email address">
```

Comment:
take name parameter from the django form
`name="{{ form.<field>.name" }}`

Also, I see that you've already defined what the field's widget will be. You can simply let Django handle the form generation.
`{{ form.field }}`

Check out this page for more usage tips: https://docs.djangoproject.com/en/1.11/topics/forms/#rendering-fields-manually

Reply:
I'll approve this for now. But want these to be changed later.

Reply:
Actually I am aware of this possibility and I made use of it in an earlier version of the register template.  However, for some strange reason I could not make use of it for the login page. I left the changes to "LoginForm" although I don't actually use it, because I think that the template  as it stands  it is rather ugly and in near future we should better change it.